dashboard/v1.1/package.json

```
@@ -6,7 +6,7 @@
     "@material-ui/core": "4.11.1",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.56",
-    "apexcharts": "^3.19.0",
+    "apexcharts": "^3.26.2",
```

```suggestion
    "apexcharts": "3.19.0",
```
@2019UCP1350 Instead of making all the changes, can you please make the version constant for the `apexcharts` package.